Formula/llvm.rb

```diff
@@ -25,6 +25,13 @@ class Llvm < Formula
       sha256 "62014ddad6d5c485ecedafe3277fe7978f3f61c940976e3e642536726abaeb68"
     end
 
+    # Fix for D89156 that broke lldb build on MacOS
+    # We apply patch D95683
```

Comment:
```suggestion
    # Fix for D89156 that broke lldb build on MacOS
    # We apply the patch from https://reviews.llvm.org/D95683
```
This will simplify finding the associated patch review. We should also probably list this patch last (i.e. after the next one), as it's the newest one.

Reply:
Sure, I've updated the revision.